path_static_roles.go

```diff
@@ -56,58 +63,17 @@ func (b *backend) pathStaticRoles() []*framework.Path {
 			HelpSynopsis:    staticRoleHelpSynopsis,
 			HelpDescription: staticRoleHelpDescription,
 		},
-	}
-}
-
-// fieldsForType returns a map of string/FieldSchema items for the given role
-// type. The purpose is to keep the shared fields between dynamic and static
-// roles consistent, and allow for each type to override or provide their own
-// specific fields
-func fieldsForType(roleType string) map[string]*framework.FieldSchema {
```

Comment:
This provided little utility, so I removed it. There was only one type being passed into it. I found the code more readable by including the fields directly into the path fields list (provides at-a-glance understanding at the API without having to jump around functions).

Reply:
I decided to add this back to minimize the diff in this PR.